install/empty_data.php

```diff
@@ -3115,7 +3115,28 @@ public function getEmptyData(): array
                 'event'        => 'add_document',
                 'is_recursive' => 0,
                 'is_active'    => 0,
-            ]
+            ], [
+                'id' => 80,
+                'name' => 'New KnowbaseItem',
+                'itemtype' => 'KnowbaseItem',
+                'event' => 'newknowbase',
```

Comment:
for each event, I suggest to append item to indicate better the concerned component